docs/cluster-configuration.md

```diff
@@ -780,14 +780,31 @@ can be applied.
 ## Workloads protection on uninstall
 
 `UninstallStrategy` defines how to proceed on uninstall when workloads (VirtualMachines, DataVolumes) still exist:
-- `BlockUninstallIfWorkloadsExist` will prevent the CR from being removed when workloads still exist. 
+- `BlockUninstallIfWorkloadsExist` will prevent the CR from being removed when workloads still exist.
 BlockUninstallIfWorkloadsExist is the safest choice to protect your workloads from accidental data loss, so it's strongly advised.
 - `RemoveWorkloads` will cause all the workloads to be cascading deleted on uninstallation.
 **WARNING**: please notice that RemoveWorkloads will cause your workloads to be deleted as soon as this CR will be, even accidentally, deleted.
 Please correctly consider the implications of this option before setting it.
 
 `BlockUninstallIfWorkloadsExist` is the default behaviour.
 
+
+## Cluster level EvictionStrategy
+
+`UninstallStrategy` defines how to proceed on uninstall when workloads (VirtualMachines, DataVolumes) still exist:
```

Comment:
Is it blong here? Also, this is a duplication of line 782.

Reply:
Removed it.